test/uk/gov/hmrc/vatsubscription/connectors/UpdateVatSubscriptionConnectorSpec.scala

```diff
@@ -64,7 +65,6 @@ class UpdateVatSubscriptionConnectorSpec extends TestUtil with MockHttpClient {
         val result: Future[UpdateVatSubscriptionResponse] = connector.updateVatSubscription(testUser, requestModel, hc)
 
         await(result) shouldBe Right(SuccessModel("12345"))
-        connector.writes shouldBe UpdateVatSubscription.DESApi1365Writes
```

Comment:
Any reason this was removed?

Reply:
It fails due to the fact that the Writes now takes an `appConfig` parameter and when the two anonymous functions are compared for equality they have two different references. Jack and I couldn't find a way to rewrite this, and ultimately after looking at exactly what it was testing it didn't seem like it was an important test (bit confused why it was checking the writes when there is only 1 writes in `UpdateVatSubscription`?)

If you think this is still a valuable test though we could try to explore other ways to rewrite it.